src/vs/workbench/contrib/files/browser/views/media/openeditors.css

```diff
@@ -37,13 +37,15 @@
 }
 
 .open-editors .monaco-list .monaco-list-row.dirty:not(:hover) > .monaco-action-bar .codicon-pinned::before {
-	content: "\ebb2"; /* use `pinned-dirty` icon unicode for sticky-dirty indication */
-	font-family: 'codicon';
+	/* use `pinned-dirty` icon unicode for sticky-dirty indication */
+	content: var(--vscode-icon-pinned-dirty-content);
+	font-family: var(--vscode-icon-pinned-dirty-font-family);
 }
 
 .open-editors .monaco-list .monaco-list-row.dirty:not(:hover) > .monaco-action-bar .codicon-close::before {
```

Comment:
`.codicon-close` should be there anymore. It would set the close icon
But we would need a rule so that all oder codicon properties are inherited:
https://github.com/microsoft/vscode/blob/94f7c1d971f71d70cdef7b7ed461f399162d08c3/src/vs/base/browser/ui/codicons/codicon/codicon.css#L12

I think we should set
`.codicon[class*='codicon-']` to `.codicon`

Need to investigat if that has unwanted effects